### ISSUE TITLE
Move to given coordinates on the Image - Question

### ISSUE THREAD

**Issue by vivsamcom** [openseadragon/openseadragon#2275]
We have following requirements wherein we have to achieve following on Image Viewer. Is this feasible ?

Clicking on Annotated Image should take the user to the same co-ordinates on the image where it was taken.
Here while annotating image we will have to save its co-ordinates.

ROI (Region of Interest) - Same as above.
Clicking on the ROI image (generated by AI) will take the user to original image at the same co-ordinates which is referred by ROI.
Here AI will also give us the co-ordinates up to certain zoom level.

Can this be achieved using method zoomTo(zoom, refPointopt, immediatelyopt) → {[OpenSeadragon.Viewport](https://openseadragon.github.io/docs/OpenSeadragon.Viewport.html)}

Which other method can be used to move to specific coordinates on the image?

Thanks in advance.

**Comment by pearcetm** [comment]
This is certainly feasible. Have a look at [Viewport.panTo](https://openseadragon.github.io/docs/OpenSeadragon.Viewport.html#panTo). You can combine that with `zoomTo` if desired.

**Comment by iangilman** [comment]
Indeed! Also, see my answer at https://github.com/openseadragon/openseadragon/discussions/2274

**Comment by vivsamcom** [comment]
Thanks a lot for your response. I will try the solutions mentioned.
